test/e2e/scripts/e2e-ci.sh

```
@@ -0,0 +1,32 @@
+echo "  Payload Processing Image: $PAYLOAD_PROCESSING_IMAGE"
+echo "  Payload Processing E2E Image: $PAYLOAD_PROCESSING_E2E_IMAGE"
+echo "================================================"
+
+echo "Checking simulator connectivity..."
+reachable=true
+
+if nc -z -w 5 "$E2E_SIMULATOR_ENDPOINT" 443 2>/dev/null; then
+    reachable=true
+    echo "Simulator is reachable"
+else
+    reachable=false
```

Shellcheck (0.11.0)

[warning] reachable appears unused. Verify use (or export if used externally).

(SC2034)